tests/src/Search/Web/DateRangeTest.php

```diff
@@ -34,13 +39,18 @@ public function test_date_range_end_only()
         ]);
 
         $this->newClient();
-        $this->jsonRequest('GET', '/search', ['end-date' => '2016-12-01']);
+        $this->jsonRequest('GET', '/search', ['end-date' => '2016-12-13']);
         $response = $this->getJsonResponse();
-        $this->assertEquals($response->total, 1);
+        $this->assertEquals(1, $response->total);
 
         $this->jsonRequest('GET', '/search', ['end-date' => '2016-10-01']);
         $response = $this->getJsonResponse();
-        $this->assertEquals($response->total, 0);
+        $this->assertEquals(0, $response->total);
+
+        // boundary
+        $this->jsonRequest('GET', '/search', ['end-date' => '2016-12-05']);
+        $response = $this->getJsonResponse();
+        $this->assertEquals(1, $response->total, 'The date upper boundary is being excluded');
```

Comment:
Could we list the IDs on the included results? Would make reading the tests easier.